packages/base/src/updateShadowRoot.js

```diff
@@ -24,7 +24,7 @@ const updateShadowRoot = (element, forStaticArea = false) => {
 		styleStrOrHrefsArr = getEffectiveStyle(element.constructor, forStaticArea);
 	}
 
-	element.constructor.render(renderResult, shadowRoot, styleStrOrHrefsArr, { host: element });
+	element.constructor.render(renderResult, shadowRoot, styleStrOrHrefsArr, forStaticArea, { host: element });
```

Comment:
Again here, it's strange to pass a `forStaticArea` parameter as `updateShadowRoot` should be just for the non-static part

Reply:
Yes. Some controls (for example ui5-select) have additional template for static area which will execute updateShadowRoot function again. This will add busy indicator markup to the template added in static are and busy indicator could be shown twice.

![image](https://user-images.githubusercontent.com/31909318/159288716-695d880a-5335-421d-893b-7c55ff472c9c.png)